code/src/java/pcgen/persistence/PersistenceManager.java

```diff
@@ -65,7 +65,7 @@ public static PersistenceManager getInstance()
 	 * 
 	 * CODE-1889 to remove use of this method
 	 */
-	public void setChosenCampaignSourcefiles(List<URI> l)
+	public static void setChosenCampaignSourcefiles(List<URI> l)
```

Comment:
I want to eliminate that, but in a separate PR. (I should have actually separated this one into two). In general I'm trying to keep my PRs smaller.